bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/Wprimetotb/Wprimetotb_LH_customizecards.dat

```diff
@@ -0,0 +1,5 @@
+set MADSPIN=ON
```

Comment:
Doesn't have to turn on madspin yourself

Reply:
But I just realised this, where is your madspin card?

Reply:
It is the same already present in the folder. It was approved in our previous commit for the RH production.

Reply:
By the way you can find it [here](https://github.com/cms-sw/genproductions/blob/master/bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/Wprimetotb/Wprimetotb_madspin_card.dat)

Reply:
@adeiorio Can you add full set of your cards that will go into gridpacks as an input?

Reply:
I have now pushed the madspin and extramodels cards, so that everything we use is here.